app/gather/assets/apps/components/PaginationBar.jsx

```diff
@@ -84,9 +84,16 @@ class PaginationBar extends Component {
     }
   }
 
-  componentWillReceiveProps (nextProps) {
-    if (nextProps.currentPage !== this.state.currentPage) {
-      this.setState({ currentPage: Math.min(Math.max(1, nextProps.currentPage), this.getNumberOfPages()) })
+  getSnapshotBeforeUpdate (prevProps, prevState) {
```

Comment:
Wouldn't it be better to use `getDerivedStateFromProps` here?

Reply:
I tried but I could not make it work :disappointed_relieved: 

I can try it harder :hammer_and_wrench:

Reply:
Weird.... it doesn't look _that_ complicated, although I've never used it before, so what do I know? 🤷‍♂

Reply:
OK, I finally manage to make it :sweat_smile: